appengine/metadata/standard/server.js

```diff
@@ -22,31 +22,32 @@ const request = require('got');
 const app = express();
 app.enable('trust proxy');
 
-const METADATA_NETWORK_INTERFACE_URL = 'http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip';
+const METADATA_PROJECT_ID_URL = 'http://metadata.google.internal/computeMetadata/' +
```

Comment:
Nit: can this be on one line? (You may have to add an `// eslint-disable` at the end of it if you get linter issues.)